src/first/Main.java

```diff
@@ -2,90 +2,40 @@
 
 import java.io.*;
 import java.util.*;
-import java.nio.file.DirectoryStream;
 
 public class Main {
-    static String extension;
     static final String PATH = "D:\\1";
+    static String extension = "txt";
```

Comment:
please, get rid of hardcoded values and use Scanner instead. And lets write test